### PR TITLE
Drop Django < 1.8 compatibility

### PR DESCRIPTION
**Pull request by jleclanche**
This is a subset of #289 which is safe to land for the next release, per the comments.

### CODE REVIEW

File: polymorphic/admin/parentadmin.py [hunk 1]
```diff
@@ -334,9 +315,9 @@ def subclass_view(self, request, path):
             try:
                 pos = path.find('/')
                 if pos == -1:
-                    object_id = long(path)
+                    object_id = int(path)
```
Comment (hobarrera) [inline polymorphic/admin/parentadmin.py]: Are you entirely sure this won't affect Python 2.7 negatively?

Comment (jleclanche) [inline polymorphic/admin/parentadmin.py]: Yes, positive.

**Comment by vdboor** [comment]
Thanks for providing this pull request!

When I have some time, I'll fix the remaining issue (or someone else should beat me first). I think the the `qrepr()` is not completely compatible with 1.8, which breaks tests, because everything else looks fine!

**Comment by jleclanche** [comment]
@vdboor any luck figuring it out?

File: polymorphic/admin/parentadmin.py (hunk 1 above)
Comment (vdboor) [inline polymorphic/admin/parentadmin.py]: I'm not so sure, as Django's `BigIntegerField` can produce longs on Python 2. I've reverted this in 67dd5c0a0dd33537c1deb946dbae0c8e98fbecb2

**Comment by vdboor** [comment]
Thanks a lot for all your work @jleclanche! :-D This work is merged now, and it should make maintenance a lot easier for new developers. 👍

**Comment by jleclanche** [comment]
Excellent! Release soon? :)

**Comment by jleclanche** [comment]
@vdboor poke poke, is the release still blocked on something? can I help with anything?

**Pull request merged**